validate.swift

```diff
@@ -322,10 +314,11 @@ func runDumpPackage(at path: URL, timeout: TimeInterval = 20) throws -> Data {
     }
 }
 
+@discardableResult
 func dumpPackage(url: URL) throws -> Package {
```

Comment:
We could just get rid of the return value here, but I left it in case it's useful in the future.